app/assets/stylesheets/lists.scss

```diff
@@ -66,7 +66,7 @@ ol#about-nav {
   li {
```

Comment:
I was able to solve this with the following css (maybe it's not the most correct solution..) :

```
ol#about-nav {
  @extend .unstyled;
  @include clearfix;
  li {
    display: inline;
    float: left;
    margin-right: 7px;
    &:last-child {
      margin-right: 0;
    }
    width: 80px;
    height: 85px;
    padding-right: 5px;
    a {
      width: 60px;
      height: 80px;
      display: block;
      opacity: 0.9;
      text-align: center;
      padding: 0 15px;
      font-size: 13px;
      line-height: 20px;
      color: $font-color;
      @include background-image-2x('/images/icons/nav-circles', 90px, 180px);
      &.current {
        @include background-image-2x('/images/icons/nav-circles', 90px, 180px, 0 -90px);
        color: #fff;
        opacity: 1;
      }
      &.one-line {
        padding-top: 35px;
        height: 55px;
      }
      &.two-line {
        padding-top: 28px;
        height: 62px;
      }
      &.three-line {
        padding-top: 17px;
        height: 73px;
      }
      &:hover {
        opacity: 1;
      }
    }
  }
}
```

Reply:
result:

![image](https://user-images.githubusercontent.com/1999050/34338970-7239237a-e966-11e7-8575-2e137119c7f0.png)

Reply:
@peff any comments on the css changes before we go for it? The circles are a bit smaller, but I think it's still readable 

![image](https://user-images.githubusercontent.com/1999050/34338962-51fdfd88-e966-11e7-9940-c43fbb0782b3.png)

Reply:
easier way to understand my css changes:

![image](https://user-images.githubusercontent.com/1999050/34339015-3df8ed56-e967-11e7-9394-ed7a5097eefd.png)

Reply:
@pedrorijo91  So shall I add the following changes to the main file and push that?

Reply:
I would say yes, just wanted to get the comments of someone else. let's see if anyone comments until tomorrow, then we proceed :)